src/app/datagrid/test-cases/test-cases.ts

```diff
@@ -21,6 +21,16 @@ export class DatagridTestCasesDemo {
     zeroUsers: User[] = [];
     pageSize: number = 7;
 
+    loading1: boolean = true;
+    loading2: boolean = false;
+    loading3: boolean = false;
+    loading4: boolean = false;
+    loading5: boolean = false;
+    loading6: boolean = false;
+    loading7: boolean = false;
+    loading8: boolean = false;
+    loading9: boolean = false;
```

Comment:
You're very patient. I would have used a single property shared on all demos, so toggling it would make them all have a spinner. 😛